pkg/user/user_integration_test.go

```diff
@@ -531,21 +521,21 @@ func TestUserHandler(t *testing.T) {
 	t.Run("AsAdmin", func(t *testing.T) {
 		t.Parallel()
 
-		var adminToken token.Tokens
+		var adminAccessToken *http.Cookie
 		{
 			t.Log("SignIn")
 
-			client.PostJSON(t, "/tokens", nil, &adminToken, inttest.WithBasicAuth("admin", "admin"))
+			adminAccessToken, _ = client.SignIn(t, "admin", "admin")
 
-			require.NotEmpty(t, adminToken.AccessToken, "should return an access token")
+			require.NotEmpty(t, adminAccessToken.Value, "should return an access token")
 		}
 
 		{
 			t.Log("GetAllUsers")
 			userCount.Wait()
 
 			var users []model.User
-			client.GetJSON(t, "/users", &users, inttest.WithAuthToken(adminToken.AccessToken))
+			client.GetJSON(t, "/users", &users, inttest.WithAuthToken(adminAccessToken.Value))
```

Comment:
Do we now always pass in a cookie? If so you could change the `WithAuthToken` or make it `WithCookie`

Reply:
We don't always pass in the a cookie. We still accept a bearer token using the authorization header